geode-junit/src/main/java/org/apache/geode/test/junit/rules/ClassAnalysisRule.java

```diff
@@ -95,19 +101,26 @@ private void loadClasses() {
             .map(x -> new File(x))
             .collect(Collectors.toList());
 
+    // check for <module>/build/classes/java/**
     String gradleBuildDirName =
-        Paths.get(getModuleName(), "build", "classes", "java", "main").toString();
-    // System.out.println("gradleBuildDirName is " + gradleBuildDirName);
+        Paths.get(getModuleName(), "build", "classes", "java", sourceSet).toString();
+
+    // check for <module>/build/classes/test/**
+    String alternateBuildDirName =
+        Paths.get(getModuleName(), "build", "classes", sourceSet).toString();
+
+    // check for <module>/out/production/classes/**
     String ideaBuildDirName =
         Paths.get(getModuleName(), "out", "production", "classes").toString();
-    // System.out.println("ideaBuildDirName is " + ideaBuildDirName);
+
+    // check for <module>/out/production/geode.<module>.<sourceSet>/**
     String ideaFQCNBuildDirName =
-        Paths.get("out", "production", "geode." + getModuleName() + ".main").toString();
-    // System.out.println("idea build path with full package names is " + ideaFQCNBuildDirName);
+        Paths.get("out", "production", "geode." + getModuleName() + "." + sourceSet).toString();
```

Comment:
Running these tests in IntelliJ uses `build/classes/java`. These special cases for IntelliJ don't seem to be used anymore. Delete them.

Reply:
Done